src/Tickets/Admin/All_Tickets/List_Table.php

```diff
@@ -0,0 +1,257 @@
+	 *
+	 * @since TBD
+	 *
+	 * @return string
+	 */
+	protected function get_primary_column_name(): string {
+		return 'name';
+	}
+
+	/**
+	 * Get hidden columns for the list table.
+	 *
+	 * @since TBD
+	 *
+	 * @return array
+	 */
+	public function get_hidden_columns(): array {
+		$screen = get_current_screen();
+		if ( is_null( $screen ) ) {
+			return $this->get_default_hidden_columns();
+		}
+		return get_hidden_columns( $screen );
+	}
+
+	/**
+	 * Returns the columns for the list table.
+	 *
+	 * @since  TBD
+	 *
+	 * @return array
+	 */
+	public static function get_default_hidden_columns(): array {
+		$default_hidden_columns = [
+			'id',
+			'start',
+			'days_left',
+			'sales',
+		];
+
+		/**
+		 * Filter the default hidden columns for the All Tickets Table.
+		 *
+		 * @since TBD
+		 *
+		 * @param array $default_hidden_columns The default hidden columns for the All Tickets Table.
+		 *
+		 * @return array
+		 */
+		return apply_filters( 'tec_tickets_all_tickets_table_default_hidden_columns', $default_hidden_columns );
+	}
+
+	/**
+	 * Returns the sortable columns for the list table.
+	 *
+	 * @since  TBD
+	 *
+	 * @return array
+	 */
+	public function get_sortable_columns() {
+		$sortable_columns = [
+			'name'      => [ 'name', true ],
+			'id'        => [ 'id', true ],
+			'start'     => [ 'start', true ],
+			'end'       => [ 'end', 'desc' ], // Start with DESC order.
+			'days_left' => [ 'days_left', true ],
+			'price'     => [ 'price', true ],
+			'sold'      => [ 'sold', true ],
+			'remaining' => [ 'remaining', true ],
+			'sales'     => [ 'sales', true ],
+		];
+
+		/**
+		 * Filters the sortable columns for the All Tickets Table.
+		 *
+		 * @since TBD
+		 *
+		 * @param array $sortable_columns The sortable columns for the All Tickets Table.
+		 *
+		 * @return array
+		 */
+		return apply_filters( 'tec_tickets_all_tickets_table_sortable_columns', $sortable_columns );
+	}
+
+	/**
+	 * Renders the default column.
+	 *
+	 * @since  TBD
+	 *
+	 * @param object $item        The current item.
+	 * @param string $column_name The current column name.
+	 *
+	 * @return string
+	 */
+	public function column_default( $item, $column_name ): string {
+		return $item[ $column_name ];
+	}
+
+	/**
+	 * Prepares the list of items for displaying.
+	 *
+	 * @since TBD
+	 */
+	public function prepare_items() {
+		$current_page = $this->get_pagenum();
+		$per_page     = $this->get_items_per_page( $this->per_page_option );
+
+		$pagination_args = [
+			'total_items' => 1,
+			'per_page'    => $per_page,
+		];
+
+		$args = [
+			'page'               => $current_page,
+			'per_page'           => $per_page,
+			'return_total_found' => true,
+			'order'              => 'DESC',
+		];
+
+		// Setup sorting args.
+		if ( tribe_get_request_var( 'orderby' ) ) {
+			$args['orderby'] = tribe_get_request_var( 'orderby' );
+		}
+
+		if ( tribe_get_request_var( 'order' ) ) {
+			$args['order'] = tribe_get_request_var( 'order' );
+		}
```

Comment:
could these be an if/elseif?

Reply:
I'm not sure I follow. These are two different variables. `orderby` would the the column key (`name`, `end`, `remaining`, etc.) and `order` is the order direction (`asc` or `desc`).